### PR TITLE
test: covered fast-style with tests

### PR DESCRIPTION
**Pull request by macseem**
Add validation for sizes enum
Added test for combine numeric enums

### CODE REVIEW

File: packages/fast-style/src/font.spec.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    TestFontSizeNumericEnum2 = 2,
+}
+
+describe('GetFont', () => {
+    it('Should create a tree for mocked font family, size and color', () => {
+        const Font = getFont(TestFontFamilyEnum, TestFontSizeEnum, TestFontColorEnum);
+        expect(Font).toEqual(expectedTree);
+    });
+    it('Should throw an error if size enum has numeric values', () => {
+        // @ts-ignore
```
Comment (vitalyiegorov) [inline packages/fast-style/src/font.spec.ts:84]: Can we avoid this ts ignore?

File: packages/fast-style/src/font.ts [hunk 2]
```diff
@@ -1,7 +1,12 @@
 import { combine, Enum } from '@rnw-community/object-field-tree';
 
-export const getFont = (fontFamilyObj: Enum, fontSizeObj: Enum, fontColorObj: Enum) =>
-    combine(
+export const getFont = (fontFamilyObj: Enum, fontSizeObj: Enum<string>, fontColorObj: Enum) => {
```
Comment (vitalyiegorov) [inline packages/fast-style/src/font.ts:3]: @macseem Change to `fontSizeObj: Enum<string | number>`

Comment (macseem) [reply packages/fast-style/src/font.ts]: changed

File: packages/fast-style/src/font.spec.ts (hunk 1 above)
Comment (macseem) [reply packages/fast-style/src/font.spec.ts]: removed

**Pull request merged**